.github/PULL_REQUEST_TEMPLATE

```diff
@@ -76,7 +76,6 @@ If you don't know who to ask, you can request reviews in https://groups.google.c
 
 #### How should this be tested?
 <!--(List steps with links to updated documentation)-->
-#### Any background context you want to provide?
```

Comment:
❓ Why? Do we often commit changes to our code base without context? Do we change our software based on a random number generator or perhaps a Magic 8-Ball :tm:?

Snark aside, I do not fundamentally understand the reasoning behind this change. Looking at the Issue for it, it seems that perhaps it too was generated by a Magic 8-Ball :tm: that hadn't yet been programmed to fill out the Issue. Huh. I guess snark isn't completely put aside after all.